docs/docs/integrations/scim.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring SCIM for GrowthBook
+description: Setting up SCIM Integration with GrowthBook
+id: scim
+slug: scim
+---
+
+# SCIM Integration for Enterprise Organizations
+
+:::note
+
+SCIM is only available with an Enterprise plan and requires [Single Sign-On (SSO)](https://docs.growthbook.io/sso) to be enabled. Currently, GrowthBook only supports Okta as the identity provider.
+
+:::
+
+SCIM, or [System for Cross-domain Identity Management](https://scim.cloud/), is the standard for managing users and groups across multiple applications. With SCIM, you can automate the provisioning and deprovisioning of users in your GrowthBook account through your identity provider.
+
+GrowthBook's SCIM integration currently offers the following features:
+
+- User provisioning
+- User deprovisioning
+
+When a user is provisioned, they are added to your GrowthBook organization with your organization's default role. After provisioning, admin users can adjust their roles and permissions via the GrowthBook application as needed. It's important to note that if a user is provisioned through SCIM, they can only be deprovisioned through your identity provider.
+
+:::note
+
+We are actively working to expand functionality to include group provisioning and deprovisioning, as well as adding support for additional identity providers.
+
+:::
+
+## Configuring SCIM Integration
+
+### Okta Setup
+
+1. Verify that your GrowthBook organization is on an enterprise plan with SSO enabled.
+
+2. Log in to your Okta account and go to the Applications page. Select "Browse App Catalog," then search for "SCIM 2.0 Test App (OAuth Bearer Token)." Click "Add Integration" to add the app to your Okta account.
+
+![](../../static/images/integrations/scim/GrowthBook-SCIM-Add-Okta-Integration.png)
+
+3. Once the app is added, you can change its name, for example, to "GrowthBook SCIM." Click "Next."
+
+![](../../static/images/integrations/scim/GrowthBook-SCIM-Add-Name.png)
+
+4. On the next page, you don't need to modify any settings. Simply click "Done."
+
+5. With the application created, click on the "Provisioning" tab and select "Configure API Integration."
+
+![](../../static/images/integrations/scim/GrowthBook-SCIM-Enable-API-Integration.png)
+
+6. Choose to enable API Integration, and input your credentials. Contact your account representative to obtain the SCIM 2.0 Base URL. You can acquire your OAuth Bearer Token by creating a new Secret API Key. To do this, go to your GrowthBook account, and in the left navigation, select "Settings > API Keys." We recommend creating a dedicated API key exclusively for your SCIM integration.
```

Comment:
Is there any reason why we can't provide the baseURL for them here? Should be the same for all cloud users and for self-hosted it should just be `{SELF_HOSTED_URL}/api/scim/v2`

Reply:
We should also mention somewhere that the user creating the API key needs to have manage team permissions